packs/classfeatures/greater-field-discovery-chirurgeon.json

```diff
@@ -24,7 +24,106 @@
             "remaster": true,
             "title": "Pathfinder Player Core 2"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "inline-healing",
+                "key": "RollOption",
+                "option": "chirurgeon-greater-field-discovery",
+                "toggleable": true
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "downgrade",
+                "predicate": [
+                    "chirurgeon-greater-field-discovery",
+                    "dice:slug:base",
+                    {
+                        "or": [
+                            "item:slug:elixir-of-life-minor",
+                            "item:slug:elixir-of-life-lesser",
+                            "item:slug:elixir-of-life-moderate",
+                            "item:slug:elixir-of-life-greater",
+                            "item:slug:elixir-of-life-major",
+                            "item:slug:elixir-of-life-true"
+                        ]
+                    }
+                ],
+                "property": "dice-number",
+                "selectors": [
+                    "inline-healing"
+                ],
+                "value": 0
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "chirurgeon-greater-field-discovery",
```

Comment:
Isn't this the item it's on?

Reply:
This is the toggleable roll option I set up earlier